conf/app.routes

```diff
@@ -2,3 +2,4 @@
 
 GET        /assets/*file            controllers.Assets.versioned(path="/public", file: Asset)
 GET        /hello-world             uk.gov.hmrc.plasticpackagingtax.registration.controllers.HelloWorldController.helloWorld
```

Comment:
Once the start page is in we should be able to remove the `/hello-world` URI

Reply:
ok, I'm currently pointing my start button to hello world controller, thought I'll leave that around until we have next page to go to. Do you want  me to remove that and just leave blank button url?

Reply:
as agreed, removed